client/index.html

```diff
@@ -1,11 +1,473 @@
+      </p>
+  </article>
+
+  <article class="Article Article--stats">
+    <h2>Display compatible browsers</h2>
+    <form id="query_form" class="Form">
+      <textarea name="query" cols="30" rows="6" required
+                class="Form__queryTextArea" id="query_text_area"
+                placeholder="Write the Browserslist config here..."></textarea>
+      <p class="Form__interactivePlaceholder">Write the Browserslist config here... <br>
+        For example
+        <a class="QueryLink" data-query="defaults"><code>defaults</code></a> ,
+        <a class="QueryLink" data-query="not dead"><code>not dead</code></a>
+      </p>
+      <p class="Form__hint Form__hint--progress">Press Enter
+        <span role="presentation" class="Form__enterIcon">
+          <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="none">
+            <path
+              d="M10.71 1.42h-.8a.1.1 0 0 0-.1.1v6.94H3.28V7.5a.1.1 0 0 0-.17-.09l-1.9 1.5a.1.1 0 0 0 0 .17l1.9 1.5a.1.1 0 0 0 .17-.08v-1h6.67c.48 0 .86-.4.86-.87v-7.1a.1.1 0 0 0-.1-.1Z"
+              fill="#1E1E1E" fill-opacity=".4"/>
+          </svg>
+        </span>
+        to submit
+      </p>
+      <p class="Form__hint Form__hint--required">Query should be not empty</p>
+      <p class="Form__hint Form__hint--error" id="error_message">Unknown query</p>
+    </form>
+    <div class="BrowsersStat__placeholder">
+      <img src="./assets/stats.svg" role="presentation">
+      <p class="BrowsersStat__placeholderText">add the Browserslist config to display compatible browsers</p>
+    </div>
+    <div class="BrowsersStat__stat BrowsersStat__stat--hidden">
+      <h2>Global coverage: <span id="global-coverage"></span></h2>
+      <ul role="presentation" id="global-coverage-bar" class="BrowsersStat__globalCoverageBar"></ul>
+      <div id="browsers-stats" class="BrowsersStat"></div>
+      <div class="ToolsVersions">
+        <p>
+          Data provided by caniuse-db <span id="can_i_use_version"></span>
+        </p>
+        <p>Browserslist version: <span id="browsers_list_version"></span></p>
+      </div>
+    </div>
+    <!-- TODO write real documentation -->
+  </article>
+
+  <article class="Article Article--docs">
+    <h2>How to get started</h2>
+    <p>
+      <a class="QueryLink" data-query="defaults"><code>defaults</code></a> - if you're building a web application for
+      the global audience.
+    </p>
+    <p>
+      <a class="QueryLink" data-query="node 18"><code>node 18</code></a> - if you're building a Node.js application,
+      e.g., for SSR.
+    </p>
+    <p><a href="" class="Link">Autoprefixer</a> Autoprefixer ,
+      <a href="" class="Link">Babel</a> or any of the following libraries will find target browsers automatically, when
+      you add the following to <b>package.json</b>:
+    </p>
+    <a class="QueryLink" data-query="defaults, not IE 11, maintained node versions"><pre class="Pre">
+      <code>"browserslist": [</code>
```

Comment:
Why do we have `code` for every line?

Reply:
Because i think i look more readable in html. And do not broke the tabs line.

Reply:
OK, let’s keep it. Anyway in we will need to refactor it when we will be moving to Markdown.